test/meetingsession/MeetingSessionConfiguration.test.ts

```diff
@@ -17,6 +17,38 @@ describe('MeetingSessionConfiguration', () => {
       expect(configuration.urls).to.be.null;
     });
 
+    it('any attribute can be null', () => {
```

Comment:
Please add unit tests for these cases too:
- `undefined` property
- An empty string property `''`
- `null` property in the child object, e.g. `MediaPlacement.SignalingUrl` is `null`
- an array of `null` and `undefined` elements, e.g. `[null, undefined, { ... }]`

You can add one more test, including all those cases